server/src/main/java/io/crate/planner/operators/LogicalPlanner.java

```diff
@@ -132,10 +137,11 @@
  * Planner which can create a {@link ExecutionPlan} using intermediate {@link LogicalPlan} nodes.
  */
 public class LogicalPlanner {
+
     private final IterativeOptimizer optimizer;
-    // Join ordering optimization rules have their own optimizer, because these rules have
+    // Join implementations optimization rules have their own optimizer, because these rules have
     // little interaction with the other rules and we want to avoid unnecessary pattern matches on them.
-    private final IterativeOptimizer joinOrderOptimizer;
+    private final IterativeOptimizer joinImplementationOptimizer;
```

Comment:
The Optimizer rules are rearranged to keep changes in the related query plans from the tests to a minimum.